lib/fake_aws/s3/responses/empty.rb

```diff
@@ -14,7 +14,7 @@ def headers
         end
 
         def body
-          ""
+          [""]
```

Comment:
`Rack` [body response](http://rack.github.io/) is supposed to be returned as an enumerable.